constellation-app/constellation
# Fix Test Notifications Ok button exception

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Exception fixed by returning set of non-null PluginParameters in CreateParameters().
Dark themed Show/Hide Details text fixed with stylesheet.



### Alternate Designs
None.


### Why Should This Be In Core?
To fix the unexpected exception on the Ok button for Test Notifications.



### Benefits
To fix the unexpected exception on the Ok button for Test Notifications.


### Possible Drawbacks
None.


### Verification Process

1. Create a new Graph.
2. Select menu item Experimental -> Developer -> Test Notifications. 
3. Click OK.

Notification dialogs pop up and no exception is displayed. All text in the dialog boxes are readable.



### Applicable Issues
#2139
